python/ray/train/tests/test_torch_predictor.py

```diff
@@ -154,7 +165,7 @@ def forward(self, input_dict: dict):
             out2 = self.linear2(input_dict["B"])
             return out1 + out2
 
-    predictor = TorchPredictor(model=CustomModule())
+    predictor = TorchPredictor(model=CustomModule(), use_gpu=use_gpu)
```

Comment:
Should we add a test to see if GPU is actually being used? Maybe just see if the model is correctly being moved to cuda.